tests/Unit/bootstrap.php

```diff
@@ -4,3 +4,4 @@
 require_once $vendor_dir . '/yoast/wp-test-utils/src/BrainMonkey/bootstrap.php';
 require_once $vendor_dir . '/autoload.php';
 require_once __DIR__ . '/MonkeyStubs.php';
+require_once __DIR__ . '/../../../../../wp-includes/class-wp-error.php';
```

Comment:
Need a clear linespace at the end of the file.

Reply:
This is not needed anymore, we can throw an exception when WP_Error is not available.